src/lib/propertiesConverters.js

```diff
@@ -9,26 +9,15 @@
 import _ from 'lodash'
 import { format } from 'd3-format'
 
-export const getLabelGenerator = (_label, labelFormat) => {
-    if (_.isFunction(_label)) {
-        return _label
-    }
-
-    const label = d => _.get(d, _label)
+export const getLabelGenerator = (_label, labelFormat = '') => {
+    if (_.isFunction(_label)) return _label
 
-    let formatter
-    if (labelFormat) {
-        formatter = format(labelFormat)
-    }
+    const formatter = format(labelFormat)
+    const labelFn = d => _.get(d, _label)
 
     return data => {
```

Comment:
With the previous changes we can offer 2 levels of customization, accessor & formatting:

```javascript
if (formatter) return d => formatter(getRawLabel(d))
return getRawLabel
```

This way, we avoid the condition to be executed for each label retrieval.